indico/modules/events/surveys/templates/emails/new_submission_email.txt

```diff
@@ -0,0 +1,11 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    New survey submission
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
```

Comment:
This email still lacks information. event title, link to the survey management page (eventually it should link to the actual submission), link to the event (could go into the `footer_url` block).